src/components/entity/ha-chart-base.html

```diff
@@ -0,0 +1,182 @@
+<link rel='import' href='../../../bower_components/polymer/polymer.html'>
+
+<dom-module id="ha-chart-base">
+  <link rel="import" type="css" href="../../../bower_components/billboard.js/dist/billboard.min.css">
+  <style>
+     :host {
+      display: block;
+      /* @apply --my-custom-chart-istic-mixin; */
+    }
+  </style>
+  <template>
+    <div id="chartTarget"></div>
+  </template>
+</dom-module>
+<script src="../../../bower_components/d3/d3.js"></script>
+<script src="../../../bower_components/d3-gtimeline/dist/d3-gtimeline.js"></script>
+<script src="../../../bower_components/billboard.js/dist/billboard.js"></script>
+<script>
+/* global d3 bb */
+
+(function () {
+  if (customElements.get('ha-chart-base')) return;
+  /**
+   * `billboard-charts`
+   * Polymer wrapper around billboard.js charting library
+   *
+   * @customElement
+   * @polymer
+   */
+  class ExtenedChart extends Polymer.Element {
```

Comment:
we should name classes after the custom element name, so in this case HaChartBase